alvr/session/src/settings.rs

```diff
@@ -201,32 +187,76 @@ Does not work with the "Reduce color banding" option, requires enabling "Use pre
 }
 
 #[derive(SettingsSchema, Serialize, Deserialize, Clone, PartialEq)]
+#[schema(collapsible)]
 pub struct SoftwareEncodingConfig {
     #[schema(strings(
         display_name = "Force software encoding",
-        help = "Forces the encoder to use CPU instead of GPU"
+        help = "Forces the server encoder to use CPU instead of GPU"
     ))]
     #[schema(flag = "steamvr-restart")]
     pub force_software_encoding: bool,
 
-    #[schema(strings(display_name = "Encoder thread count"))]
+    #[schema(strings(display_name = "CPU encoder thread count"))]
```

Comment:
should be "Software encoder thread count"

Reply:
Might need to rename it in the code though, not sure if leaving it as software in code a good idea